src/App.tsx

```diff
@@ -1,28 +1,69 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { FilterStatusType, Todo } from './types/Todo';
+import * as tadoService from './api/todos';
+import classNames from 'classnames';
+import { Footer } from './component/Footer';
+import { Error } from './component/Error';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+
+function todoInUse(todo: Todo[], filter: FilterStatusType) {
+  const todoUsed = [...todo];
+
+  switch (filter) {
+    case FilterStatusType.All:
+      return todoUsed;
+    case FilterStatusType.Active:
+      return todoUsed.filter(todoItem => todoItem.completed === false);
+    case FilterStatusType.Completed:
+      return todoUsed.filter(todoItem => todoItem.completed === true);
```

Comment:
```suggestion
      return todos.filter(todo => !todo.completed);
    case FilterStatusType.Completed:
      return todos.filter(todo => todo.completed);
```
- `item` in a name is redundant
- You don't need to compare with true/false as it's already boolean